data/HoldUpFjord.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "Dominick J Monaco",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Conservation Tech turned Software Engineer. Interested in contributing to community, climate change solutions, and dinner parties",
+  "avatar": "https://github.com/HoldUpFjord.png",
+   "tags": [
+    "100Devs",
+    "Green-tech",
+    "Javascript",
+    "Node.js",
+    "Tailwind",
+    "React",
+    "MongoDb",
```

Please remove this trailing comma `,` from here. This is leading `json` check to fail.

```suggestion
    "MongoDb"
```

---

This should be fixed as well!